cmd/dlv/cmds/commands.go

```diff
@@ -409,9 +410,7 @@ func dapCmd(cmd *cobra.Command, args []string) {
 		}
 		defer logflags.Close()
 
-		if headless {
-			fmt.Fprintf(os.Stderr, "Warning: dap mode is always headless\n")
-		}
+		headless = true
```

Comment:
I don't think this temp var is used anywhere except when initializing the `debugger.Config` below so let's just get rid of it and set the value to `true` below.

Reply:
Heh, full circle to what the Foreground was before we started messing with it :) Done.